Makefile

```diff
@@ -363,9 +376,9 @@ push-operator: prepare-csv
 
 ## Push-Image: push container image to upstream, including latest tag.
 push-image: build-image
-	podman tag "$(OPERATOR_IMAGE):$(OPERATOR_TAG_LONG)" "$(OPERATOR_IMAGE):latest"
-	-podman push "$(OPERATOR_IMAGE):$(OPERATOR_TAG_LONG)"
-	-podman push "$(OPERATOR_IMAGE):latest"
+	$(CONTAINER_RUNTIME) tag "$(OPERATOR_IMAGE):$(OPERATOR_TAG_LONG)" "$(OPERATOR_IMAGE):latest"
```

Comment:
travis CI conf should be updated because it depends on docker. Or to make `docker` as default?

Reply:
good point... let's switch to `docker` as default